config.json.ctmpl

```diff
@@ -0,0 +1,4 @@
+{
+    "dosResolutionHost": {{if eq (env "ENVIRONMENT") "prod"}}"dcp.bionimbus.org"{{else}}"dcp.bionimbus.org"{{end}},
+    "bondBaseUrl": {{if eq (env "ENVIRONMENT") "prod"}}"broad-bond-prod.appspot.com"{{else if eq (env "ENVIRONMENT") "staging"}}"broad-bond-staging.appspot.com"{{else if eq (env "ENVIRONMENT") "alpha"}}"broad-bond-staging.appspot.com"{{else}}"broad-bond-dev.appspot.com"{{end}}
```

Comment:
if you put `{{with $environment := env "ENVIRONMENT"}}` at the top of the file and `{{end}}` at the bottom you can simplify this line to:

`"bondBaseUrl": "broad-bond-{{$environment}}.appspot.com"`